astor/rtrip.py

```diff
@@ -85,6 +85,12 @@
 dsttree = 'tmp_rtrip'
 
 
+def out_prep(s):
+    if b'' == u'':
+        return s
+    return s.encode('UTF-8')
```

Comment:
Yeah, sometimes I miss things.  I thought we were going to run flake8 or something as part of the regression.  Is there a reason why that's problematic?